jekyll/_cci2/artifacts.md

```diff
@@ -142,9 +142,9 @@ curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/:build_n
 <artifacts.txt xargs -P4 -I % wget %?circle-token=$CIRCLE_TOKEN
 ```
 
-Note 1: Replace all the variables above that start with a : with real values for your project (don't include the colon).
+Note 1: Replace all the variables above that start with a *:* (colon) with real values for your project (don't include the colon).
 
-Note 2: :vcs-type will be github or bitbucket.
+Note 2: `:vcs-type` will be `github` or `bitbucket`.
```

Comment:
Backticks are an excellent addition! You could also combine this note into the previous as an example of the replacement. Up to you!